Altis_Life.Altis/core/functions/fn_startLoadout.sqf

```diff
@@ -6,76 +6,23 @@
     Description:
     Loads a custom loadout on player when he got a new life
 */
-private _pUniform = M_CONFIG(getArray,"Loadouts",str(playerSide),"uniform");
-private _pVest = M_CONFIG(getArray,"Loadouts",str(playerSide),"vest");
-private _pBackpack = M_CONFIG(getArray,"Loadouts",str(playerSide),"backpack");
-private _pWeapon = M_CONFIG(getArray,"Loadouts",str(playerSide),"weapon");
-private _pMagazines = M_CONFIG(getArray,"Loadouts",str(playerSide),"mags");
-private _pItems = M_CONFIG(getArray,"Loadouts",str(playerSide),"items");
-private _linkedItems = M_CONFIG(getArray,"Loadouts",str(playerSide),"linkedItems");
-
-if !(_pUniform isEqualTo []) then {
-    if (playerSide isEqualTo civilian) then {
-        _pUniform = selectRandom _pUniform;
-        if (!(_pUniform isEqualTo []) && {!((_pUniform select 0) isEqualTo "") && {([(_pUniform select 1)] call life_fnc_levelCheck)}}) then {
-            player forceAddUniform (_pUniform select 0);
-        };
-    } else {
-        _pUniform apply {
-            if (!(_x isEqualTo []) && {!((_x select 0) isEqualTo "") && {([(_x select 1)] call life_fnc_levelCheck)}}) then {
-                player forceAddUniform (_x select 0);
-            };
-        };
-    };
-};
-
-if !(_pVest isEqualTo []) then {
-    _pVest apply {
-        if (!(_x isEqualTo []) && {!((_x select 0) isEqualTo "") && {([(_x select 1)] call life_fnc_levelCheck)}}) then {
-            player addVest (_x select 0);
+private _level = call {
+    if (playerSide isEqualTo civilian) exitWith {
+        if (life_is_arredted) then {
+            "arrested"
+        } else {
+            str(floor random [0, round(count(missionConfigFile >> "Loadouts" >> str(playerSide)-1)/2), count(missionConfigFile >> "Loadouts" >> str(playerSide))-1])
         };
     };
-};
-
-if !(_pBackpack isEqualTo []) then {
-    _pBackpack apply {
-        if (!(_x isEqualTo []) && {!((_x select 0) isEqualTo "") && {([(_x select 1)] call life_fnc_levelCheck)}}) then {
-            player addBackpack (_x select 0);
-        };
+    if (playerSide isEqualTo west) exitWith {
+        str(FETCH_CONST(life_coplevel))
```

Comment:
instead of using str around each of the levels (for civ, cop and med), an easier solution could be to use format on line 25:
`player setUnitLoadout (missionConfigFile >> "Loadouts" >> str(playerSide) >> format["lvl_"_level];`